src/provider/bpmn/BpmnPropertiesProvider.js

```diff
@@ -206,17 +206,23 @@ function getGroups(element) {
 
 export default class BpmnPropertiesProvider {
 
-  constructor(propertiesPanel) {
+  constructor(propertiesPanel,translate) {
+    BpmnPropertiesProvider.prototype.translate=translate
     propertiesPanel.registerProvider(this);
   }
 
   getGroups(element) {
+    const translate=this.translate
     return (groups) => {
       groups = groups.concat(getGroups(element));
+      // translate group name
+      groups.map(function(group){
```

Comment:
Minor thing: Why do we use `map` instead of `forEach`? We are not mapping anything here into a new array.